src/styles/stars.css

```diff
@@ -0,0 +1,17 @@
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+
+  background-image: url("../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(5) {
+  background-image: url("../images/star.svg");
+}
+
```

Comment:
Look to p.4 of checklist.  You have to reuse part of Stars task here, and keep it operational in case you change block modifier.  As I understood, star--0, star--1... star--5 in our task with stars are  the modifiers of our block stars. And they must be here.  Then in case you change modifier in your index.html, you'll have right number of active stars.